client/landing/gutenboarding/stores/plans/reducer.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import * as plans from '../../../../lib/plans/constants';
+import { getPlan } from '../../../../lib/plans';
+
+const supportedPlanSlugs = [
+	plans.PLAN_FREE,
+	plans.PLAN_PERSONAL,
+	plans.PLAN_PREMIUM,
+	plans.PLAN_BUSINESS,
+	plans.PLAN_ECOMMERCE,
+];
+
+const supportedPlans = supportedPlanSlugs.map( getPlan );
+
+import { Plan, PlanAction } from './types';
+
+const DEFAUlT_STATE: { selectedPlan: Plan; supportedPlans: Array< Plan > } = {
+	supportedPlans,
+	selectedPlan: getPlan( supportedPlanSlugs[ 0 ] ),
```

Comment:
We should leave it `undefined` unless there is explicitly set by using `/beginner` route or using the PlansGrid.